c10/util/string_view.h

```diff
@@ -15,579 +15,120 @@
 
 namespace c10 {
 
-/**
- * Port of std::string_view with methods from C++20.
- * Implemented following the interface definition in
- * https://en.cppreference.com/w/cpp/string/basic_string_view
- * See there for the API documentation.
- *
- * Difference: We don't have a Traits template parameter because
- * std::char_traits isn't constexpr and we'd have to reimplement
- * std::char_traits if we wanted to use it with our constexpr basic_string_view.
- */
```

Comment:
Can we have an updated comment for what this class does ontop of the std version?

Reply:
Updated